person.rb

```diff
@@ -1,12 +1,16 @@
+require './corrector'
+
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] Since a person has many rentals and can create rentals, getters and setters for @rentals should be present

Reply:
well solved